templates/main.php

```diff
@@ -167,46 +167,46 @@ class="collection"
             <div class="app-navigation-entry-menu bubble sortorder-dropdown">
```

Comment:
Hm, I can't reproduce this behavior. After selecting an option I can open and close the menu just fine. It behaves the same as before. Could you maybe create an issue for that and tell me which browser you use?